docs/software/installing/easybuild.md

```diff
@@ -175,6 +258,53 @@ the EasyBuild recipe follows the EasyBuild guidelines for naming. If the
 guidelines are not followed and if EasyBuild needs to install this module as a
 dependency of another package, EasyBuild will fail to locate the build recipe.
 
+The `GROMACS/2021.4-cpeGNU-22.08-PLUMED-2.7.4-CPU` module can now be used just like
+any other module on the system. To use the GROMACS module you don't need to load `EasyBuild-user`, 
+so all you need to do to use that module when running with this module (which we installed
+in `partition/C` of `LUMI/22.08` in this example) is
```

Comment:
There is no earlier mention of switching to partition/C in this guide

Reply:
Changed the text in a new commit.